dns/cloudflare.go

```diff
@@ -105,13 +106,20 @@ func (cf *Cloudflare) addUpdateDomainRecords(recordType string) {
 			return
 		}
 
+		params := domain.GetCustomParams()
+		// 存在参数才进行筛选
+		var commentParam string
+		if params.Has("comment") {
+			commentParam = fmt.Sprintf("&comment=%s", params.Get("comment"))
+		}
+
 		zoneID := result.Result[0].ID
 
 		var records CloudflareRecordsResp
 		// getDomains 最多更新前50条
 		err = cf.request(
 			"GET",
-			fmt.Sprintf(zonesAPI+"/%s/dns_records?type=%s&name=%s&per_page=50", zoneID, recordType, domain),
+			fmt.Sprintf(zonesAPI+"/%s/dns_records?type=%s&name=%s&per_page=50%s", zoneID, recordType, domain, commentParam),
```

Comment:
感谢您的澄清。

```suggestion
			fmt.Sprintf(zonesAPI+"/%s/dns_records?type=%s&name=%s&per_page=50&comment=%s", zoneID, recordType, domain, domain.GetCustomParams().Get("comment")),
```

另外，鉴于 URL 参数值可以为空，直接拼接字符串即可。

Reply:
这里参数为空的意思是“筛选出comment为空的记录”，而预期是“不筛选记录”，因此采用了现在的写法